### PR TITLE
feat: add compose view button

### LINKED ISSUE #9: Show Compose button

### PR DESCRIPTION
**Pull request by pyphilia**
This PR adds the compose button in the top right corner + the current item name in the top left corner.

close #9 
Screenshot 2021-06-28 at 17 35 32

### CODE REVIEW

**Comment by pyphilia** [comment]
I've extended this PR with the user information in the header. The new layout is the following
Screenshot 2021-06-29 at 11 11 23

File: src/utils/user.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line import/prefer-default-export
+export const isRegularUser = (user) => {
+  if (!user) {
+    return false;
+  }
+
+  // todo: to change
+  return !user.get('email').includes('@graasp.org');
```
Comment (juancarlosfarah) [inline src/utils/user.js:8]: Indeed this looks like it's bound to break. What's the actual logic going to be like?

File: src/config/constants.js [hunk 2]
```diff
@@ -32,6 +33,14 @@ export const AUTHENTICATION_HOST =
...
   'http://localhost:3112';
 
+export const GRAASP_COMPOSE_HOST =
+  ENV_GRAASP_COMPOSE_HOST ||
+  process.env.REACT_APP_GRAASP_COMPOSE_HOST ||
+  'http://localhost:3111';
+
 // define a max height depending on the screen height
 // use a bit less of the height because of the header and some margin
 export const SCREEN_MAX_HEIGHT = window.innerHeight * 0.8;
```
Comment (juancarlosfarah) [inline src/config/constants.js:43]: We might want to calculate this exactly. I.e. innerHeight minus header size.

**Review by juancarlosfarah — APPROVED** [review]
Nice! Approving. Just some questions and minor comments.

File: src/utils/user.js (hunk 1 above)
Comment (pyphilia) [reply src/utils/user.js:8]: I'm not sure, currently light users are identical to regular users, except their automatically generated email as well as some extra data they have. Probably checking their additional data will be more accurate.

File: src/config/constants.js (hunk 2 above)
Comment (pyphilia) [reply src/config/constants.js:43]: Actually the header size is never defined and depends on the content. And probably this screen size height is better defined depending on the height alone, as it let some margins to the elements. If you still want to depend on the header height, we can open an issue for that.

**Pull request merged**